### DISCUSSION TITLE
Add radius- server deadtime in eos_cli_config_gen

### DISCUSSION THREAD

**Discussion by chrisgerdener** [aristanetworks/avd#4151] [Data Model Schemas]
Hello everyone,

We are wanting to configure global deadtime for our radius servers on our fabric and did not immediately see a way to do this in AVD yet. So I wanted to ask if I am just missing something and this functionality is already covered by AVD. 

In case it is not I want to propose a change to the data model for the radius_server, which would be to add a new key called deadtime under which a duration can be configured.

```
radius_server:
  deadtime: 
    duration: <int>
```
    
What do you think?

**Comment by ClausHolbechArista** [comment]
Hi Chris, apologies for missing your request. I don't see this covered, so please submit an enhancement request for the CLI you need and the suggested model. Please close this discussion afterwards so we don't get duplicates. Thanks.
